src/NzbDrone.Core/Configuration/ConfigService.cs

```diff
@@ -148,6 +148,13 @@ public bool RemoveFailedDownloads
             set { SetValue("RemoveFailedDownloads", value); }
         }
 
+        public bool UpdateSeriesIfFolderMissing
```

Comment:
I think `Update` is the wrong term here, we're going to upgrade regardless (fetch info from skyhook), but we're going to "scan" it. `ScanMissingSeriesFolders` would be better I think.